Update fullnode.md (#3797)
MystenLabs/sui: doc/src/build/fullnode.md

Please see our [privacy policy](https://sui.io/policy/) to learn how we handle
information about our nodes.

> ## Typical isues 

Reviewer: ```suggestion
## Resolving errors 
```

> 1. Error 98/99  ![Error 98](https://user-images.githubusercontent.com/86624298/183159157-46971a06-a5c4-4196-80b0-63177a78164d.png) 

Reviewer: ```suggestion
#### Error 98/99
```

> You should restart your server ```shell $ shutdown -r now  ``` and after that will run Restart your Sui fullnode  ```shell
>     $ cargo run --release --bin sui-node -- --config-path fullnode.yaml
>     ```

Reviewer: ```suggestion
If you receive an error with code 98 or 99, you should restart your server using the following command:
`$ shutdown -r now` 

Then, restart your Sui fullnode:
`$ cargo run --release --bin sui-node -- --config-path fullnode.yaml`
```

Is there an extra -- in the command?
--bin sui-node -- --config-path
